cogs/fun.py

```diff
@@ -110,6 +143,39 @@ class KarmaEmotes(enum.Enum):
                     await database.update_karma(reaction.message.author.id,
                                                 (0, 1 if increment else -1))
 
+    # Check if the message obtained enough karma to get forwarded to another channel
+    async def forward_message_check(self, reaction: discord.Reaction):
+        message: discord.Message = reaction.message
+        if self.enabled and not message.author.bot \
+                and reaction.count >= database.settings.get_das_mooi_threshold() \
+                and message.id not in self.forwarded_messages:
+            emoji: discord.emoji.Emoji = reaction.emoji
+            if type(emoji) == discord.emoji.Emoji and (emoji.name == self.KarmaEmotes.POSITIVE.value
+                                                       or emoji.name ==
+                                                       self.KarmaEmotes.NEGATIVE.value):
+                self.forwarded_messages.append(message.id)
+                channel = self.bot.get_channel(
+                    database.settings.get_das_mooi_channel()) \
+                    if emoji.name == self.KarmaEmotes.POSITIVE.value \
+                    else self.bot.get_channel(database.settings.get_das_niet_mooi_channel())
+                await channel.send(
+                    f"A new message by {message.author.mention} arrived:\n"
+                    f"> {message.content}\n"
+                    f"> \n"
+                    f"> {message.jump_url}\n",
+                    files=await to_files(message.attachments))
```

Comment:
message.attachments is a property that returns a list of Assets if i recall, so if you want the raw data from it, just use https://discordpy.readthedocs.io/en/latest/api.html#discord.Attachment.read